INSTRUCTION.md

```diff
@@ -0,0 +1,12 @@
+## HOW shoude TO USE
```

Comment:
The phrase 'HOW shoude TO USE' contains a typographical error. It should be corrected to 'HOW TO USE'.